app/templates/search.html

```diff
@@ -0,0 +1,171 @@
+                    <span class="badge badge-danger ml-2">
+                      Bị xóa
+                    </span>
+                  {% elif post.mode == 1 %}
+                    <span class="badge badge-dark ml-2">
+                      Trả phí 
+                    </span>
+                  {% endif %}
+                </div>
+              </div>
+
+              <div class="d-flex">
+                {% for hashtag in post.hashtags.all %}
+                  <div class="bg-secondary text-white px-1 mr-1 mt-1" data-toggle="tooltip" data-placement="bottom" title="Hashtag">{{ hashtag.name }}</div>
+                {% endfor %}
+              </div>
+
+              <div class="d-flex mb-0 mt-1">
+                <div data-toggle="tooltip" data-placement="bottom" title="Lượt xem">
+                  <i class="fa-solid fa-eye text-muted"></i>
+                  <span class="text-muted">{{ post.view_count }}</span>
+                </div>
+                <div class="ml-2" data-toggle="tooltip" data-placement="bottom" title="Số lượng Bookmark">
+                  <i class="fa-regular fa-bookmark"></i>
+                  <span class="text-muted">{{ post.bookmark_count }}</span>
+                </div>
+                <div class="ml-2" data-toggle="tooltip" data-placement="bottom" title="Số lượng bình luận">
+                  <i class="fa-regular fa-comment"></i>
+                  <span class="text-muted">{{ post.comment_count }}</span>
+                </div>
+                <div class="points ml-2" data-toggle="tooltip" data-placement="bottom" title="Tổng đánh giá">
+                  <div class="carets">
+                    <i class="fa-solid fa-caret-up fa-sm mb-1"></i>
+                    <i class="fa-solid fa-caret-down fa-sm"></i>
+                  </div>
+                  <span class="text-muted">{{ post.feedback_value }}</span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="border"> </div>
+      {% endfor %}
+    {% elif choices_single_default == "Author" %}
+      <div class="row mt-5">
+        {% for author in object_list %}
+          <div class="media col-4 mb-4">
+            <img class="d-flex align-self-center mr-3 rounded-circle mb-3" src="{{ author.avatar_link }}" width="50" height="50" alt="Avatar">
+            <div class="media-body">
+              <p class="my-0"><a href="{% url 'profile' author.username %}">
+                {{author.first_name}} {{author.last_name}}
+                {% if author.is_superuser %}
+                  <span class="badge badge-danger">Admin</span>
+                {% elif author.count_violated == 3 %}
+                  <span class="badge badge-danger">Banned</span>
+                {% elif author.is_staff %}
+                  <span class="badge badge-warning text-white">Moderator</span>
+                {% elif not author.is_active %}
+                  <span class="badge badge-secondary">Not Activate</span>
```

Comment:
1. Translate đống text, nhớ gõ makemessage
2. Chỗ kết quả hiển thị search Author, tớ nghĩ chỉ nên hiển thị những người đang hoạt động thui, ko cần những người Not Active đâu (như Facebook lúc tài khoản bị khóa/xóa thì tìm cũng ko ra ý)

Reply:
Not Activate mà

Reply:
à, Not Activave thì cũng ko cần thiết, ngta chưa kích hoạt thì cũng coi như chưa đăng ký thành công, chưa có bài viết nào, show ra hình như cũng không dùng tới ý
Cơ mà để bên admin thì được